shared/chat/conversation/messages/wrapper/exploding-meta/index.js

```diff
@@ -111,56 +91,57 @@ class ExplodingMeta extends React.Component<Props, State> {
 
   render() {
     const backgroundColor =
-      this.props.explodesAt - Date.now() < oneMinuteInMs ? globalColors.red : globalColors.black_75
+      this.props.explodesAt - Date.now() < oneMinuteInMs
+        ? Styles.globalColors.red
+        : Styles.globalColors.black_75
     let children
     switch (this.state.mode) {
       case 'countdown':
-        let bombIconSize = isMobile ? 22 : 16
-        if (isAndroid) {
-          // icon is 24 high and clips edge of container on android. workaround
-          bombIconSize = 21
-        }
+        const bombIconSize = Styles.isMobile ? 21 : 14
         children = (
-          <Box2 direction="horizontal" gap="xtiny">
+          <Kb.Box2 direction="horizontal" gap="xtiny">
             {this.props.pending ? (
-              <Box2 direction="horizontal" style={styles.progressContainer}>
-                <ProgressIndicator style={{height: 12, width: 12}} />
-              </Box2>
+              <Kb.Box2 direction="horizontal" style={styles.progressContainer}>
+                <Kb.ProgressIndicator style={{height: 12, width: 12}} />
+              </Kb.Box2>
             ) : (
-              <Box2
+              <Kb.Box2
                 direction="horizontal"
-                style={collapseStyles([
+                style={Styles.collapseStyles([
                   styles.countdownContainer,
                   {
                     backgroundColor,
                   },
                 ])}
               >
-                <Text type="Body" style={styles.countdown}>
+                <Kb.Text type="Body" style={styles.countdown}>
                   {formatDurationShort(this.props.explodesAt - Date.now())}
-                </Text>
-              </Box2>
+                </Kb.Text>
+              </Kb.Box2>
             )}
-            <Icon type="iconfont-bomb" fontSize={bombIconSize} color={globalColors.black_75} />
-          </Box2>
+            <Kb.Icon type="iconfont-bomb" fontSize={bombIconSize} color={Styles.globalColors.black_75} />
+          </Kb.Box2>
         )
         break
       case 'boom':
         children = (
-          <Box2 direction="horizontal" style={styles.boomIconContainer}>
-            <Icon
+          <Kb.Box2 direction="horizontal" style={styles.boomIconContainer}>
```

Comment:
This doesn't seem to show on explode anymore (this might also be busted on master)

Reply:
yeah this happens on master and its something about the timing logic (not styling) in react tools if i force state "boom" then it works fine